packages/edit-site/src/components/sidebar-global-styles-wrapper/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useMemo, useState } from '@wordpress/element';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+import { useViewportMatch } from '@wordpress/compose';
+import {
+	Button,
+	privateApis as componentsPrivateApis,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import GlobalStylesUI from '../global-styles/ui';
+import Page from '../page';
+import { unlock } from '../../lock-unlock';
+import StyleBook from '../style-book';
+import { STYLE_BOOK_COLOR_GROUPS } from '../style-book/constants';
+
+const { useLocation, useHistory } = unlock( routerPrivateApis );
+const { Menu } = unlock( componentsPrivateApis );
+const GLOBAL_STYLES_PATH_PREFIX = '/wp_global_styles';
+
+const GlobalStylesPageActions = ( {
+	isStyleBookOpened,
+	setIsStyleBookOpened,
+} ) => {
+	return (
+		<Menu
+			trigger={
+				<Button __next40pxDefaultSize variant="secondary">
```

Comment:
```suggestion
				<Button __next40pxDefaultSize variant="tertiary" size="compact">
```

Looks like we're missing one of the final tweaks proposed in https://github.com/WordPress/gutenberg/pull/65619#issuecomment-2451585626.

> The PR here is already virtually there, it's just a matter of updating the style of the preview button. Since we use tertiary blue link buttons in similar contexts, I'd go with such a one, here, as well, in its compact size.